### ISSUE TITLE
Test failures with non-English locale

### ISSUE THREAD

**Issue by eserte** [cazador481/MooX-Ipc-Cmd#1]
Test suite of 1.0.0 fails with non-English locale (e.g. LC_ALL=de_DE.UTF-8):

```
Running make test
PERL_DL_NONLAZY=1 "/opt/perl-5.18.2/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
t/00-compile.t ........... ok
"our" variable $VERSION redeclared at /opt/perl-5.18.2/lib/site_perl/5.18.2/MooX/Log/Any.pm line 3.
t/03_signal.t ............ ok
"our" variable $VERSION redeclared at /opt/perl-5.18.2/lib/site_perl/5.18.2/MooX/Log/Any.pm line 3.
sh: 1: [/opt/perl-5.18.2/bin/perl: not found

#   Failed test 'failed capture'
#   at t/04_capture.t line 57.
#                   'Datei oder Verzeichnis nicht gefunden'
#     doesn't match '(?^:No such file or directory)'
# Looks like you failed 1 test of 9.
t/04_capture.t ...........
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/9 subtests
"our" variable $VERSION redeclared at /opt/perl-5.18.2/lib/site_perl/5.18.2/MooX/Log/Any.pm line 3.

#   Failed test 'failed capture'
#   at t/05_multi_capture.t line 64.
#                   'Datei oder Verzeichnis nicht gefunden'
#     doesn't match '(?^:No such file)'
# Looks like you failed 1 test of 13.
t/05_multi_capture.t .....
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/13 subtests
        (less 2 skipped subtests: 10 okay)
"our" variable $VERSION redeclared at /opt/perl-5.18.2/lib/site_perl/5.18.2/MooX/Log/Any.pm line 3.
t/09_system.t ............ ok
t/author-critic.t ........ skipped: these tests are for testing by the author
t/release-unused-vars.t .. skipped: these tests are for release candidate testing
```

**Comment by eserte** [comment]
This is the case for all perl versions between 5.10 and 5.20 on linux and freebsd, but not for 5.21, because $! is not anymore locale-specific in the latter version.

**Comment by cazador481** [comment]
Thank you for the bug report.  I will hopefully have a fix this weekend.

**Comment by eserte** [comment]
Thanks, seems to be fixed with 1.0.1.
